src/test/java/seedu/guest/logic/parser/ParserUtilTest.java

```diff
@@ -35,6 +37,7 @@ public class ParserUtilTest {
     private static final String VALID_NUMBER_OF_GUESTS = "4";
     private static final String VALID_IS_ROOM_CLEAN = "yes";
     private static final String VALID_BILL = "10.00";
+    private static final String VALID_REQUEST = "I want to book for room service";
```

Comment:
Should we change it to "May I have room service?"

Reply:
changed